build.sbt

```diff
@@ -418,9 +418,12 @@ def crossPublishLocal(scalaV: String) = Def.task[Unit] {
       ),
       state.value
     )
-  Project
+  val (s, _) = Project
     .extract(newState)
     .runTask(publishLocal.in(mtags), newState)
+  Project
+    .extract(s)
+    .runTask(publishLocal.in(interfaces), s)
```

Comment:
@olafurpg Travis is complaining about a missing `mtags-interfaces`, which I guess is normal now that the optimization about transitive dependencies is not in place anymore?

I've added this to run `interfaces/publishLocal` in addition to `mtags/publishLocal` before running slow tests